protractor.config.js

```diff
@@ -0,0 +1,5 @@
+exports.config = {
+	framework: 'jasmine',
+	seleniumAddress: 'http://localhost:4444/wd/hub',
+	specs: ['test/spec.js']
+}
```

Comment:
add empty line at the end of each file, this red icon show that there is no empty line at the end of the file, you can check or use editor confi for solve this problem in your text editor.
![image](https://user-images.githubusercontent.com/2055110/39060079-f3bad2a2-4485-11e8-9566-0642a935a6a9.png)